apps/molgenis-components/src/components/display/ImageDisplay.vue

```diff
@@ -48,24 +48,32 @@ export default {
zCCoQ8sW0IAn+kETQ48JLBlMAhCYHjaZx6zF8ZCBwPs9RiaiwpheAfVAj+tZHgn+MJqXBeKoHL5tlazF8BBK5HZE9t6QhGKYJmx1GLzR+BBA0N6JgxnA44m4nj5Ri14pSWjv2jEZRaH6/HsoHUs9HgGt+iaoERtZiCTSfQggpqBA0PKoi1SPZ6bM1Gd4/0oIim+KZNbo4M4YoNBIvm1iNT1GtT1NZ9U3XNGP/dGGt50ExXMMeLNhI0L+I4Nz06K0jQTGkGbjUBoqaPm4cAXZwGxd7vN82bwYsowPsPGhaxGO2gjMJ3UGybhIBmYD5z7gFp0FiBePANUTkEo8bEfyfQ3QcFBK0VaQYCQXm3veXe6NMMhHyAgluVmoBQAFB0s9VoVneLRLyCCwOLbhcb1dyoolnRPXXFd9Tt6lRnPCjMn8U9gTVezIKnxdmzuauxtm5e4e1utgStxbFuTM41At+kVJqglgsnbW8ttb03tn/qmuRbuEApgtYSn7JZJYYrSw4laI1O2Kza3VZaloDtvThNMZCc+9wOI6DFcApvwDvbi4fLE1BvIMeuShhZe4ATwbFvc0clLq92JWitno52HTTxu6UqkCNBqzXZHYWB+APoLsCqCVqt9a7yukTzX+IC94MJaDl0K2VAfnftmBNXglbruo8r6vAR7PctLquunIDaeFZBQSA8G9srcMUEtDL18UEQCOO+a/WphoBab4odexs0+1OrIcCRCagtOp5fkgJh3+u7NT4Zq4SA2vX5lFAKG9cmNPdoOnb33bRVRcBstdzvsE/rFDBPRmje6d/vOsuqcs+sSgJuq8W4090in+U1ZT7adjvjRfluQ4FVThDZarLoLc/HzM6Xy8Wk8pxHdjSCk9kzQf32TFC/PRPUb88E9dszQf32TFC/PRPUb88E9dszQf32TFC/PRPUb88E9dszQf1WnsB6xeiQpaUSz1MIxk/T6fRJXZW4Zj+ZKqvs5zvs+/62o8ye7+nv6PsIxiLhPpnv5U9P7Ik/Ys1fGSd6Sl7Wpz9X1ndG+y19Ht4ZtowoBH2MEEod1ln69CcoPv2yIGLxkiA/OQzQxQhnt8MtUJww/iv+9MRE6B7+guRohI/ivY1bmiQ5PDf25QMx0VbdVAK2jkTU7ZGd1E96vljGYP/Eu5iAxFFsYgCxA4YnjPdJZE5vY07A4yYll5qj5GVbohz/Gypv1s6yZAlYxNDYeKi+6KE8JAfyZwPMXht/d52An9NmCRH7Q7TdjxEQZT1kHRMkJ6YBAh7YB/vbrY8zGTQTKHsb+POjh7K/xn32sh5RXqsTsONseBoljI4TUwLydD2JbZQQxFkFCNiHFFWNxQ7M1lmdIKl91ykCnByQZ3EhotqrExARdzRKKINVUQKkHaURBCQ6bGkmGClv4+EPCwiU1yxVAhaBMy7uKYpfqxGwMo+DGdCE8pvlEMiLkCEClo3o4nb2Hf30gpZGoLhyR/3BGCsut0w+hUZA/y45iE7/Q5ZHHkEUi8hMMETKsWr6OpxeRc8QbJMPwuofoVVQPnSv5oBFgsQAAUONz+DSZklyQwSEFlNU6cwENF8y8Jt4TOaIcpqA0P8nrky/Xyd+KMtBrBXsWjh5xlAjGMafXSRUft9EgM9n8VczE0xTBBhldkKmCdA5ThCpG+JzfHwCWrqyYsBlEP/meb/fz6tFmL4xTk3rMJ4cn2C8jt8J+AHWIjzkENCaE5+h3SMyuD6UwKdKVEDQitsaMwFvBeEdeGkCn6lH5Mo7gjqHEiSWQ8Ca3AVMwOJmJs1gIQGrlLLzxOroobXIiqCH5aMBAtaqQydjdQLxTGYr6siTkxC0InUBCHhHC9wMnyEY0XzL91NH9lunIejKNhzqm/IYGr4WqcJMQHs10vGpI2+rJSCDobT9KEVA22ze4kMEIvaBv9ceaiToItn9oY7cr5aAjnii8U2agPbd+GcDCaiQsc6HHm7DREB7IWIQwbWtWoLYMgS8BbzOIxCDHOPxOo1gIR/KHHlddRnE45sMAe1LsS5sDgH1BTbYIPpGK41gJF2ZPTyrqYf6QXchLeoX9JLPxYQ0j6A1YaNMwzFTjaDlifiz1JFnhxGMesIW8e9DbRHPMO325xKII6JEmx7SCabiX13C8nEIwURuFSzsVYhfon/IJxCxTPIiMEkCVicXrETZsw8iEMOXwp6doGUdhyICHjrAz8y36AQL7lXCkQ8uA2RJ0OKDhEICFjOQZNpUnYC58hN7NvuPQwhWw+E4PULKIWD9saVXRMDjyKbD2RkIeFFRWeAac4rxAf8THSSQp5mRgG1ajTLNCgGn57wMBPzfwpFPRyAjV5kItlQ/4onamdZNNRBwV5YJT0eQmSlRCAb0jzEBVW84pmBEwEeasm8B5IARyIGQaa4iafHWuGiuIiJYYGuCwjLgYyIvplHmi5JJlGQSSiNIJgdEFiVOEUELJOgiZbp3pnm4iWDA6yTvYycEPXXmbJi0exrBGitzTvt4pqSQYIoAgg5KErHBDk4LgomAV8lYleRD+exlRO/Ftw8Y5k2x4mwonrwuJODzgNCsYzRXMVQ/D0zAI6hLrUyqBJ9BFvgstlE0e8AI0hWThfSTxc4Syrm9QgJ+EQY00pdVkVeETLfCRMC7A6LVVQh4bC+/Oxz3EVKC+7H7l2eREe47POFOJoxaDj4+iBPOvGgFJCFgTzISsBEaIp3xkJ/xzk68mAi4U8Uw8UP5OIkgfmQoicfDb5Am0sQvqQnjNSyxHhSbrxOwojdrMg+9gsQpHy2mnZGAr920sgStfnygFCfBVLqGsVeckOBtRGpchVqqBMzTzASjWXQqmvhaTVQIpr4vc9ChfTJRSyZsRTJOsdjxDrPvKa67Y2uM/H/s/zK/i4HPUvqzxEP6URKZmnvUkv4pSTJQXjajSRT3GnvizTv9AFjJ9eTRerlc2JxkGi1owirP3qzoA9emJeX/A0azrvJNMdx4AAAAAElFTkSuQmCC"></image-display>
```

Comment:
maybe put the image in a separate file so this file is a bit more readable?

Reply:
yes, but not a part of the current pr